### ISSUE TITLE
Consider always including 0 in the default views of charts.

### ISSUE THREAD

**Issue by chihuahua** [tensorflow/tensorboard#553]
Charts that start at values above 0 can mislead the unwary eye.

It seems as if accuracy increased by a lot here.
![sabktxz4nbz](https://user-images.githubusercontent.com/4221553/30677901-78d2aad0-9e43-11e7-81d4-1507ba7d7032.png)

This looks like a big difference in dropout rate.
![j0er0jgzb3s](https://user-images.githubusercontent.com/4221553/30677933-a056d766-9e43-11e7-91e6-f05fff7b16c7.png)

This looks volatile.
![c3j2sj3abxu](https://user-images.githubusercontent.com/4221553/30677944-b79555c4-9e43-11e7-91e0-7955b706cac6.png)

**Comment by dandelionmane** [comment]
I once did something similar, although it privileged the domain [0, 1] - it was something like, if all the data is in [0, 1], set the domain to [0, 1]. People complained, and I rolled it back, but I think the issue with my fix was actually including 1 - b/c it made comparing small learning rates (1E-4 vs 1E-5 and such) nigh impossible. Just privileging 0 might be more popular. 

Before you make this change, I encourage you to think a lot about what kind of charts need to show small differences that don't include 0, and might be rendered unusable if you force 0 into the display. Maybe ask some researchers.

**Comment by chihuahua** [comment]
SG, I'll ponder more and ask around.

Basically, Edward Tufte (champion of graphical integrity) would like this.
http://davidgiard.com/2011/05/10/DataVisualizationPart3GraphicalIntegrity.aspx

**Comment by wchargin** [comment]
Counterpoint: https://qz.com/418083/its-ok-not-to-start-your-y-axis-at-zero/

My reading of VDQI didn't suggest to me that Tufte insisted that all charts start at zero, but I'll give it another read in the next few days and see if I'm misremembering.

**Comment by wchargin** [comment]
In case you don't read the article (which is worth reading), I'll pull one quote:

> Edward Tufte, an expert in data presentation, agrees. “In general, in a time-series, use a baseline that shows the data not the zero point,” [he wrote on his website](https://www.edwardtufte.com/bboard/q-and-a-fetch-msg?msg_id=00003q), “don’t spend a lot of empty vertical space trying to reach down to the zero point at the cost of hiding what is going on in the data line itself.”

**Comment by chihuahua** [comment]
Wow, thanks. Basically, I extrapolated Tufte's point about bar charts starting at 0. I can see why line charts are different because comparison between bars is a different use case.

**Comment by dandelionmane** [comment]
As a further +1 on the developing consensus to not force 0 - your first example is an accuracy plot with the subtitle "it looks like accuracy increased a lot here". One thing I notice in reading the DL literature is that a difference of 2% can easily be the delta between "unimpressive" and "state of the art" - so an increase in accuracy from 86% to 98% does merit the full range of the y axis :)

**Comment by chihuahua** [comment]
Ah k. Makes sense.
